scRNA-seq/README.md

```diff
@@ -7,9 +7,13 @@ It covers normalization and dimension reduction methods that can be used for bot
 
 The notebooks that comprise this module are (estimated time to complete notebook):
 
-* [QC and filtering of single cell data](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/master/scRNA-seq/01-filtering_scRNA-seq.nb.html) (1 hr 20 minutes)
-* [Normalization using scater/scran](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/master/scRNA-seq/02-normalizing_scRNA-seq.nb.html) (1 hr)
-* [Quantification of tag-based data](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/master/scRNA-seq/04-tag-based_scRNA-seq_processing.nb.html) (35 minutes)
-* [Dimension reduction](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/master/scRNA-seq/05-dimension_reduction_scRNA-seq.nb.html) (1 hr 20 minutes)
+* [Quantification of tag-based scRNA-seq data](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/master/scRNA-seq/01-scRNA_quant_qc.nb.html)
```

Comment:
This links to `master`. We have GH pages on. No need to use HTML preview here! Which is good because GH pages has a better outcome.